lwip/README.md

```diff
@@ -0,0 +1,25 @@
+# LWIP Native Example
+
+Assumes there is an existing STM project with native LwIP support. wolfSSL needs compiled with the macro `WOLFSSL_LWIP_NATIVE`
+
+The main function to call in the example code is `tls_echoclient_connect();`
+
+Macros in `lwipopts.h` were also adjusted to better handle TLS packets, which can be as large as 16k. The macros adjusted were:
+- MEM_SIZE(16*1024)
+- MEMP_NUM_PBUF (10 -> 20)
+- PBUF_POOL_SIZE (8 -> 9)
+- PBUF_POOL_BUFSIZE (512 -> 2048)
+- TCP_WND (2*TCP_MSS -> 11*TCP_MSS)
+
+
+To start up a server for the example client to connect to use the bundled wolfSSL example
+server on a desktop:
+
+```
+cd wolfssl
+./configure && make
+./examples/server/server -b -d -p 49151 -c ./certs/server-ecc.pem -k ./certs/ecc-key.pem -i
+```
+
+If you have the `DEBUG_PORT` macro defined and `DEBUG_WOLFSSL` then also start up a TCP server for listing for debug messages:
```

Comment:
listing -> listening      

Change 

"If you have the `DEBUG_PORT` macro defined and `DEBUG_WOLFSSL`"

to 

"If you have the `DEBUG_PORT` and `DEBUG_WOLFSSL` macros defined "